.github/dependabot.yml

```diff
@@ -11,3 +11,5 @@ updates:
       - "cebe"
     schedule:
       interval: "daily"
+    ignore:
+      update-types: ["version-update:semver-patch"]
```

Comment:
Wouldn't this ignore bug fixes for updates that we have already applied? That seems a bit risky to me.

Reply:
Sorry don't quite understand: "bug fixes for updates that we have already applied"?

Reply:
Let's say we update a minor version of a library from 1.1.3 to 1.2.0 (this update should not be ignored according to the setting). If the maintainers of the library then realise that they introduced a bug with this change, they will release a patch version for the library, something like 1.2.1.

AFAIU the `update-types` setting above would prevent us from getting this patch update, potentially leaving us with a broken library in use (until the next minor version update).

Does this explanation make sense?